src/transformers/training_args_tf.py

```diff
@@ -22,7 +22,9 @@ class TFTrainingArguments(TrainingArguments):
     )
     mode: str = field(
         default="sequence-classification",
-        metadata={"help": 'Type of task, one of "sequence-classification", "token-classification" '},
+        metadata={
+            "help": 'Type of task, one of "sequence-classification", "token-classification", "question-answering"'
```

Comment:
(not related to this PR) 

we should probably change "sequence-classification" to "text-classification" to be consistent with:

- pipelines
- labels on the website

Reply:
Done!